jenkins/fpm.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+version=2.46.1
+
+fpm -v ${version} \
+    --iteration 1.el7 \
+    --epoch 1 \
+    --vendor EDINA \
+    --provides jenkins \
+    -a noarch \
+    -m john.pinto@ed.ac.uk \
```

Comment:
Might be a good idea to include java as a dependency? You can list the dependencies using the -d option to fpm